src/extension.ts

```diff
@@ -1,18 +1,23 @@
 import * as vscode from "vscode";
 import { ConfigValue } from "./configValue";
 import { Controller } from "./controller";
+import { resolveVsCodeGlobalVariables } from "./resolveVsCodeGlobalVariables";
 import { TestRunner } from "./runner";
 import { SourceMapStore } from "./source-map-store";
 
 export async function activate(context: vscode.ExtensionContext) {
   const smStore = new SourceMapStore();
   const includePattern = new ConfigValue("include", ["${workspaceFolder}"]);
   const excludePatterns = new ConfigValue("exclude", ["**/node_modules/**"]);
+  const nodeJsParametersRaw = new ConfigValue("nodejsParameters", []);
+  const nodejsParameters = nodeJsParametersRaw.value.map((p) => resolveVsCodeGlobalVariables(p));
```

Comment:
The value should only be accessed when tests are actually run, otherwise changes to the config value won't be picked up until the editor is reloaded

Reply:
ok doing it now right before the `spawnWorker` - is that a good place?